chat-plugins/info.js

```diff
@@ -202,7 +203,11 @@ exports.commands = {
 				}
 			});
 		}
-		if (!results.length) return this.errorReply("No results found.");
+		if (!results.length) {
+			let invalidTarget = net.isIP(target.trim()) === 0 && !target.includes('.');
```

Comment:
This simplifies to `!target.includes('.')` - you really don't need the `net` dependency.